src/main/scala/li/cil/oc/integration/railcraft/ModRailcraft.scala

```diff
@@ -14,5 +14,6 @@ object ModRailcraft extends ModProxy {
 
     Driver.add(new DriverBoilerFirebox)
     Driver.add(new DriverSteamTurbine)
+    Driver.add(DriverAnchor)
```

Comment:
This seems inconsistent with the 2 lines above it, does it work as coded?

Reply:
First 2 drivers are written in Java, hence the difference